src/MeetingsSDKAdapter.js

```diff
@@ -66,7 +67,8 @@ const EXIT_CONTROL = 'leave-meeting';
 const AUDIO_CONTROL = 'mute-audio';
 const VIDEO_CONTROL = 'mute-video';
 const SHARE_CONTROL = 'share-screen';
-const ROSTER_CONTROL = 'member-roster';
+
+export const ROSTER_CONTROL = 'member-roster';
```

Comment:
does it need to be exported?

Reply:
No, this export is no longer needed because now I use the 'member-roster' in the test file.